modules/@apostrophecms/doc-type/index.js

```diff
@@ -2703,8 +2703,9 @@ module.exports = {
               subquery.limit(undefined);
               subquery.page(undefined);
               subquery.perPage(undefined);
-              const mongo = await subquery.toMongo();
-              const count = await mongo.count();
+              // TODO is toMongo needed here?
+              // const mongo = await subquery.toMongo();
```

Comment:
It is, because of the lateCriteria feature which is important for certain edge cases, and also for consistency with any extensions that are created. More importantly, why did you need to change it out — is a feature missing?

Reply:
You can't add a reference to the collection to the cursor objects as they are created?